### PR TITLE
admin v3 start screen - add create logic in view screen and add new route

### PR DESCRIPTION
**Pull request by denisdimitrov**
pending on #1190

### CODE REVIEW

**Comment by denisdimitrov** [comment]
https://github.com/sparkletown/sparkle/pull/1190#discussion_r592244657 Leaving this here to avoid getting it lost.

File: src/pages/Admin/Details/Form/DetailsForm.styles.ts [hunk 1]
```diff
@@ -29,13 +29,82 @@ export const InputContainer = styled.div`
...
+  }
 
     &:focus {
       outline: none;
     }
+
+  }
+`;
+
+export const BannerContainer = styled.div`
```
Comment (mike-lvov) [inline src/pages/Admin/Details/Form/DetailsForm.styles.ts:66]: If this file is going to be used in V3, we should move the styles into scss

**Review by mike-lvov — COMMENTED** [review]
This PR is in a draft mode and @denisdimitrov said it needs polish, will make a final pass review after those are done

**Comment by denisdimitrov** [comment]
Closing the PR because it's outdated and both the structure of admin v3 and the design has changed drastically. The effort to resolve the conflicts and redesign it would be more than just re-writing the logic.

**Pull request closed without merging**